src/main/java/org/opensearch/security/dlic/rest/api/AccountApiAction.java

```diff
@@ -88,7 +88,12 @@ public AccountApiAction(Settings settings,
 
     @Override
     public List<Route> routes() {
```

Comment:
It is already default implementation, no need to override the default.

Reply:
It is not the default implementation as `routes()` is an abstract method in `BaseRestHandler` class.

Reply:
`BaseRestHandler` should not overwrite the default implementation of `RestHandler`.

Reply:
fixed

Reply:
@afazel @saratvemulapalli I refer to overwrite in `BaseRestHandler` in opensearch.

Reply:
Sure definitely, please upstream the change along with the static prefix handler it should be a very small change.

Reply:
https://github.com/opensearch-project/OpenSearch/pull/889

Reply:
Done